src/compute-client/src/protocol/command.rs

```diff
@@ -73,18 +73,18 @@ pub enum ComputeCommand<T = mz_repr::Timestamp> {
     /// [Initialization Stage]: super#initialization-stage
     InitializationComplete,
 
-    /// `AllowWrites` informs the replica that it can transition out of the
+    /// `AllowWrites` informs the replica that it can transition a collection out of the
     /// read-only computation stage and into the read-write computation stage.
```

Comment:
Since `AllowWrites` is now not a global state change but a collection state change, I don't think the concept of a "read-only computation stage" makes sense anymore. We'll have read-only collections and read-write collections running at the same time.

I think we should drop the "read-only computation stage" and instead explain `AllowWrites` in terms of the collection lifecycle. Like we are explaining `Schedule` today.